### PR TITLE
Nonsense

### PR DESCRIPTION
**Pull request by beck**
Testing review process.

### CODE REVIEW

File: nonsense/sup.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+console.log("whats");
```
Comment (beck) [inline nonsense/sup.js]: Single Comment!

File: nonsense/sup.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+console.log("up");
```
Comment (beck) [inline nonsense/sup.js]: Review! Combine this with previous.

Comment (beck-fake) [inline nonsense/sup.js]: Combine!

File: nonsense/sup.js [hunk 3]
```diff
@@ -0,0 +1 @@
+console.log("whats up");
```
Comment (beck-fake) [inline nonsense/sup.js]: add punctuation

Comment (beck-fake) [inline nonsense/sup.js]: What?

Comment (beck-fake) [inline nonsense/sup.js]: So confusing

**Comment by beck** [comment]
Nope, not changing.

**Pull request merged**